src/index.js

```diff
@@ -0,0 +1,178 @@
+/* eslint-disable no-loop-func */
+import './style.css';
+import theCheck from './modules/checkbox.js';
+
+let toDoList = [];
+
+function CreateTask(description, completed, id) {
+  this.description = description;
+  this.completed = completed;
+  this.id = id;
+}
+const pushTask = () => {
+  toDoList = JSON.parse(localStorage.getItem('toDoList')) || [];
+  const taskValue = document.querySelector('#entry').value;
+  const id = toDoList.length;
+  const completed = false;
+  const theTask = new CreateTask(taskValue, completed, id);
+  if (taskValue !== '') {
+    toDoList.push(theTask);
+  }
+};
+
+const store = () => {
+  localStorage.setItem('toDoList', JSON.stringify(toDoList));
+};
+
+const container = document.querySelector('#sub-container');
+const createTask = () => {
+  toDoList = JSON.parse(localStorage.getItem('toDoList')) || [];
+  for (let x = 0; x < toDoList.length; x += 1) {
```

Comment:
It would be better to simplify this loop and avoid repeating similar blocks of code.